services/provider/server/server.go

```diff
@@ -358,7 +358,9 @@ func (s *OCSProviderServer) GetDesiredClientState(ctx context.Context, req *pb.G
 			}
 			kubeResource.GetObjectKind().SetGroupVersionKind(gvk)
 			sanitizeKubeResource(kubeResource)
-			response.KubeResources = append(response.KubeResources, mustMarshal(kubeResource))
+			bytes := mustMarshal(kubeResources)
```

Comment:
There is a possibility of `bytes` var shadowing `bytes` module.

Reply:
renamed